src/app/shared-component/dropdown/dropdown.component.html

```diff
@@ -0,0 +1,18 @@
+						
+<div *ngIf="selected" class="status">
+	<span 
+		class="label {{selected.active_class}}" 
+		(click)="toggle()"
+		>
+		
+		{{selected.option}}
+	</span>
+	<ul *ngIf="show" class="drawer">
```

Comment:
I am keeping them as pf-dropdown as of now, because anyway we are going to change it over patternfly integration.